api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -366,7 +367,13 @@ private User initializeUserIfNeeded() {
   }
 
   private ResponseEntity<Profile> getProfileResponse(User user) {
-    return ResponseEntity.ok(profileService.getProfile(user));
+    Profile profile = profileService.getProfile(user);
```

Comment:
This breaks account creation:

```
org.pmiops.workbench.exceptions.ExceptionAdvice serverError: org.springframework.beans.factory.BeanCreationException (ExceptionAdvice.java:45)
org.springframework.beans.factory.UnsatisfiedDependencyException: Error creating bean with name 'nihApi' defined in class path resource [org/pmiops/workbench/firecloud/FireCloudConfig.class]: Unsatisfied dependency expressed through method 'nihApi' parameter 0; nested exception is org.springframework.beans.factory.BeanCreationException: Error creating bean with name 'endUserApiClient' defined in class path resource [org/pmiops/workbench/firecloud/FireCloudConfig.class]: Bean instantiation via factory method failed; nested exception is org.springframework.beans.BeanInstantiationException: Failed to instantiate [org.pmiops.workbench.firecloud.ApiClient]: Factory method 'fireCloudApiClient' threw exception; nested exception is java.lang.NullPointerException
```

Translation: there is no FC "end user client" (which NihApi requires) at this point during account creation, since the user is not logged in at that point.